Widgets/Widgets/OnThisDayWidget.swift

```diff
@@ -211,7 +211,8 @@ extension OnThisDayEntry {
             let latestEventYear = allEvents.first?.yearString,
             let article = previewEvents.first?.articlePreviews?.first,
             let year = previewEvent.year?.intValue,
-            let eventsCount = contentGroup.countOfFullContent?.intValue
+            let eventsCount = contentGroup.countOfFullContent?.intValue,
+            let previewEventIndex = allEvents.firstIndex(of: previewEvent)
```

Comment:
It's probably near-impossible for this to fail but, just in case I think you should keep it out of the guard and if it's nil, default to the index-less contentURL below, otherwise use the one with an index. It would just be a shame if the deep link url was the only reason a TimelineEntry wasn't instantiated.

Reply:
Good point - on it!

Reply:
Done. And one of the few times I'm comfortable with force unwrapping is for a URL from a static string. Though even then it makes me a tad uneasy. Let me know if you'd prefer it gone.